build_tools/docker/docker_run.sh

```diff
@@ -103,6 +103,10 @@ function docker_run() {
       --env SANDBOX_BASE=/dev/shm
     )
 
+    DOCKER_RUN_ARGS+=(
+      --privileged
+    )
```

Comment:
We're not using Docker for sandboxing, so this seems fine, but please add a comment why this is necessary and why we think it's OK to set.

Reply:
Added